src/main/java/Controller/LoginController.java

```diff
@@ -0,0 +1,27 @@
+package Controller;
+
+import InputBoundary.loginInputBoundary;
+import InputData.loginInputData;
+
+
+/**
+ * Controller for login and register use cases.
+ */
+public class LoginController {
+    private final loginInputBoundary loginInteractor;
+    private final loginInputData loginInputData;
```

Comment:
If you recall from lecture (I don't blame you if you don't it was ages ago), Input/OutputData objects are meant to be temporary.